src/opusrtp.c

```diff
@@ -787,6 +787,72 @@ void write_packet(u_char *args, const struct pcap_pkthdr *header,
   }
 }
 
+int extract(const char* input_file)
```

Comment:
This function, along with references to it and the `--extract` option need to be protected by `#ifdef HAVE_PCAP`. That's what the travis failure is about, and we don't want to require libpcap for all of opus-tools to build, so this needs to remain optional code.